.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
```

Comment:
OK ! So this file is added to add a new hook that will be run each time someone try to commit. But how git knows that he has to read this file before commiting ?

Reply:
Aaaaah my bad I didn't read the pre-commit doc before commenting. But so we need to do `pre-commit install` on our project in order to install the hook ?

Reply:
Exactly 👍